brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerResponseAdapter.java

```diff
@@ -18,12 +18,8 @@ public HttpServerResponseAdapter(HttpResponse response)
 
     @Override
     public Collection<KeyValueAnnotation> responseAnnotations() {
-        int httpStatus = response.getHttpStatusCode();
-
-        if ((httpStatus < 200) || (httpStatus > 299)) {
-            KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create("http.responsecode", String.valueOf(httpStatus));
-            return Arrays.asList(statusAnnotation);
-        }
-        return Collections.EMPTY_LIST;
+        KeyValueAnnotation statusAnnotation = KeyValueAnnotation.create(
```

Comment:
I think you misunderstood me, maybe... I agree that we should move to using the consistent keys. I was just explaining why we (inadvertently) did it the "wrong way" in the first place ;-)

I think this PR should be updated, so that it also moves to using the consistent keys.